src/System.Xml.XPath.XmlDocument/tests/Common/FileHelper.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Reflection;
+
+namespace XPathTests.Common
+{
+    public static class FileHelper
+    {
+        public static Stream CreateStreamFromFile(string xml)
+        {
+            var xmlPath = Utils.ResourceFilesPath + xml;
+            Stream s = typeof(FileHelper).GetTypeInfo().Assembly.GetManifestResourceStream(xmlPath);
```

Comment:
I'm having trouble getting the tests to pass. Many of them expect to be able to load one of the xml files located under the `TestData` folder.

This is the method that is supposed to load them: [GetManifestResourceStream](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.assembly.getmanifestresourcestream?view=net-8.0&devlangs=csharp&f1url=%3FappId%3DDev17IDEF1%26l%3DEN-US%26k%3Dk(System.Reflection.Assembly.GetManifestResourceStream)%3Bk(SolutionItemsProject)%3Bk(SolutionItemsProject)%3Bk(DevLang-csharp)%26rd%3Dtrue#system-reflection-assembly-getmanifestresourcestream(system-type-system-string)). It would return a Stream that represents one of the resourcefile.xml files, which according to the documentation, should've been embedded to the assembly, not sure how (I didn't find any magical property in the test csproj in the old repo).

The remarks indicate that the stream would be null if the resource is private, from another assembly, and does not have the appropriate reflection permission set. This does not seem to be the problem here.

`Utils.ResourceFilePath` is the prefix that is added to the the xml filename, which is set to the hardcoded value `"System.Xml.XPath.XmlDocument.Tests.TestData."`.

Reply:
@dotnet/area-system-xml do any of you know how to get this test to work?

Reply:
@tarekgh told me via chat: I need to add the xmls using <EmbeddedResource>. I naively assumed that I had to add them using <None>.